### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by natalia-matash**
Changed wrong classes and styled it

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+* {
+  box-sizing: border-box;
+}
+
+.bigBar {
+  margin: 20px 0;
+}
+
+.input {
+  font-family: "Averin", sans-serif;
```
Comment (SanyaBratashchuk) [inline src/style.css:24]:
```suggestion
  font-family: inherit;
```
Apply font-family to the `html` tag selector

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,69 @@
+@font-face {
+  font-family: "Averin";
```
Comment (SanyaBratashchuk) [inline src/style.css:2]:
```suggestion
  font-family: "Avenir";
```
Fix it everywhere

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]

File: src/style.css (hunk 2 above)
Comment (natalia-matash) [reply src/style.css]: All have fixed

**Comment by natalia-matash** [comment]
All have fixed

**Review by agniya-i — APPROVED** [review]

**Pull request closed without merging**